_posts/2017-08-21-elasticsearch-loader-0.10.0-released/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "Elasticsearch Loader 0.10.0 released"
+title-short: Elasticsearch Loader 0.10.0
+tags: [elasticsearch, kinesis, aws, nsq]
+author: Enes
+category: Releases
+---
+
+We are thrilled to announce version 0.10.0 of the Snowplow Elasticsearch Loader, our application for writing Snowplow enriched events and more to Elasticsearch.
+
+In this post, we will cover: 
+
+1. [NSQ support](/blog/2017/08/21/elasticsearch-loader-0.10.0-released#nsq-support)
+2. [Support for writing raw JSONs](/blog/2017/08/21/elasticsearch-loader-0.10.0-released#json)
+3. [Support for "AT_TIMESTAMP" as initial position](/blog/2017/08/21/elasticsearch-loader-0.10.0-released#at-timestamp)
+4. [Configuration changes](/blog/2017/08/21/elasticsearch-loader-0.10.0-released#config)
+5. [Contributing](/blog/2017/08/21/elasticsearch-loader-0.10.0-released#contributing)
+
+<!--more-->
```

Comment:
this tag should be above `In this post, we will cover: ` since the list is not rendered in the abstract anymore